src/transformers/generation/utils.py

```diff
@@ -33,6 +34,7 @@
     MODEL_FOR_SEQ_TO_SEQ_CAUSAL_LM_MAPPING,
     MODEL_FOR_SPEECH_SEQ_2_SEQ_MAPPING,
     MODEL_FOR_VISION_2_SEQ_MAPPING,
+    AutoTokenizer,
```

Comment:
This one should be `from ..models.auto.tokenization_auto import AutoTokenizer` instead